gradle/libs.versions.toml

```diff
@@ -71,7 +71,7 @@ slf4j = "1.7.36"
 spectator-atlas = "1.8.13"
 spring5 = "5.3.39"
 spring6 = "6.2.7"
-spring-javaformat = "0.0.43"
+spring-javaformat = "0.0.45"
```

Comment:
Upgrades to the latest formatter with fixes for the jSpecify annotations.

Reply:
fyi: https://github.com/micrometer-metrics/micrometer/pull/6273